src/commands/users/kaokaiToday.ts

```diff
@@ -0,0 +1,127 @@
+import axios from 'axios'
+import cheerio from 'cheerio'
+
+import { CommandHandlerConfig } from '../../types/CommandHandlerConfig.js'
+import { CommandInteraction } from '../../../node_modules/discord.js/typings/index.js'
```

Comment:
This should import from discord.js and not node_modules. Please fix.

Reply:
✅